build.ps1

```diff
@@ -34,3 +35,54 @@ else {
     Invoke-Expression -Command  "dotnet run"
     if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode)  }
 }
+
+if ($MsiGenBranches.Contains($env:APPVEYOR_REPO_BRANCH)) {
```

Comment:
So if someone names their branch `v3.x_hotfix` this logic would still trigger.

Reply:
Its okay if this is intentional, You can use `-split ","` and compare exact values

Reply:
Fixed so it's a string[].contains(), not string.contains().